src/index.js

```diff
@@ -19,12 +21,27 @@ exports.fetchManifest = async () => {
                             reject(e);
                         }
                     } else {
-                        reject(new Error(`Status: ${res.statusCode}\n${json}`));
+                        reject(new Error(`Url: ${url}\n${json}`));
                     }
                 })
                 .on('error', (err) => reject(err));
         }).on('error', (err) => reject(err));
     });
 };
 
+exports.fetchManifest = async () => {
+    const manifest = await fetchResource(MANIFEST_URL);
+
+    try {
+        const consoleReadmeSuffix = await fetchResource(CONSOLE_README_SUFFIX_URL);
+        const localReadmeSuffix = await fetchResource(LOCAL_README_SUFFIX_URL);
```

Comment:
This won't work, the `fetchResource` currently tries to parse the result as JSON, and throws if it fails.

The `fetchResource` should just return the response body as string, and then we'd either parse it as JSON in case of the manifest, or just keep it in case of these Markdown files.

Reply:
Oh thx complete forgot about that